Specs/spec-main.js

```diff
@@ -58,6 +58,8 @@
 
         /*global jasmineRequire,jasmine,exports,specs*/
 
+        Cesium.RequestScheduler.prioritize = false;
```

Comment:
A lot of the specs fail if prioritization is enabled because it lags behind a frame. So I added this here for simplicity sake, but it's probably a bad place...